### PR TITLE
Add refget proxy k8s manifests.

### PR DESCRIPTION
**Pull request by Mehrnaz-Charkhchi**
## Description
Replace GAA refget with refget proxy in review branches to be able to retrieve genomic sequences from ENA refget.

## Related JIRA Issue(s)
[JIRA
](https://www.ebi.ac.uk/panda/jira/browse/EWB-182)

## Deployment URL(s)
http://refget-proxy-review-template.review.ensembl.org

## Views affected
None
Affects only CI/CD pipeline

### CODE REVIEW

**Review by azangru — APPROVED** [review]
~I trust that the yaml file is correct.~

~Please remember to merge `refget-proxy-manifests` into `wp-k8s-review` and undo the change in line 17 of the yaml file.~

Well, trust but verify :-) There is still an error when requesting a genomic sequence:

image

Steps to replicate:

UI:
- select a species and go to genome browser page
- open the drawer
- click on the "sequence" dropdown to view the sequence
- the genomic sequence fails to load

Direct url:
GET http://refget-proxy-review-template.review.ensembl.org/api/refget/sequence/a5437debe2ef9c9ef8f3ea2874ae1d82?start=32315086&end=32400268 — 404 error

**Review by azangru — CHANGES_REQUESTED** [review]
Please make sure that refget proxy can fetch genomic sequences.

**Comment by veidenberg** [comment]
The review app is now working and I [updated the ingress](https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests/-/blob/refget-proxy-manifests/ensembl_refget_proxy_ingress.yaml#L6) to reflect the fix. This PR needs the following changes before merging:
1. Merge `refget-proxy-manifests` branch to `wp-k8s-review` branch
2. Revert branch name back to `wp-k8s-review` in `.setup-review-remplate.yaml`

File: gitlab-ci-templates/.setup-review-template.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
   before_script:
     - git clone https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests.git
-    - git -C ensembl-k8s-manifests/ checkout wp-k8s-review
+    - git -C ensembl-k8s-manifests/ checkout refget-proxy-manifests
```
Comment (veidenberg) [inline gitlab-ci-templates/.setup-review-template.yaml:17]: Please revert this change before merging.

**Review by veidenberg — APPROVED** [review]
The review app is now working and I [updated the ingress](https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests/-/blob/refget-proxy-manifests/ensembl_refget_proxy_ingress.yaml#L6) to reflect the fix. This PR needs the following changes before merging:
- Merge `refget-proxy-manifests` branch to `wp-k8s-review` branch in the [manifests repo](https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests)
- Revert the branch name back to `wp-k8s-review` in `.setup-review-remplate.yaml`

**Comment by Mehrnaz-Charkhchi** [comment]
> The review app is now working and I [updated the ingress](https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests/-/blob/refget-proxy-manifests/ensembl_refget_proxy_ingress.yaml#L6) to reflect the fix. This PR needs the following changes before merging:
> 
> * Merge `refget-proxy-manifests` branch to `wp-k8s-review` branch in the [manifests repo](https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests)
> * Revert the branch name back to `wp-k8s-review` in `.setup-review-remplate.yaml`

@veidenberg Could you please let me know where are the ingress files? I want to prevent the feature deployments from GAA refget.

**Comment by veidenberg** [comment]
@Mehrnaz-Charkhchi the ingress that I updated and is used in this PR is [here](https://gitlab.ebi.ac.uk/ensembl-web/ensembl-k8s-manifests/-/blob/refget-proxy-manifests/ensembl_refget_proxy_ingress.yaml#L6).

**Pull request merged**